test/export/testing.py

```diff
@@ -2,6 +2,196 @@
 import unittest
 from unittest.mock import patch
 
+import torch
+
+aten = torch.ops.aten
+
+_COMPOSITE_OPS_THAT_CAN_BE_PRESERVED_TESTING_ONLY = [
```

Comment:
i would still comment in the code that this list isn't required to be complete, but is convenient for getting "pretty good" test coverage or something (someone might look at it later and wonder if it's bad that an op is missing from the list)